adcm_client/objects.py

```diff
@@ -301,29 +277,31 @@ def config(self, full=False):
 
     @allure_step("Save config")
     def config_set(self, data):
+        # this check is incomplete, cases of presence of keys "config" and "attr" in config
+        # are not considered
         if "config" in data and "attr" in data:
-            # We are in a new mode with full_info == True
             if data["attr"] is None:
-                data["attr"] = []
-            history_entry = self._subcall(
-                'config', 'history', 'create',
-                config=data["config"],
-                attr=data["attr"],
-            )
-            return history_entry
+                data["attr"] = {}
+            history_entry = self._subcall('config', 'history', 'create', **data)
```

Comment:
these changes resulted in an error
`{ParameterError}{'date': 'Unknown parameter.', 'id': 'Unknown parameter.'}`
`config(full=True)` returns object with extra fields that are not needed to update the config